### PR TITLE
Disable prompt while update zsh after 13 days

### PR DESCRIPTION
**Pull request by MalikSiraj**

### CODE REVIEW

File: debian-config-jobs [hunk 1]
```diff
@@ -916,6 +916,8 @@ unset selection
 			sed -i "/^export ZSH=.*/a export ZSH_CACHE_DIR=~\/.oh-my-zsh\/cache" /etc/skel/.zshrc
 			# define theme
 			sed -i 's/^ZSH_THEME=.*/ZSH_THEME="risto"/' /etc/skel/.zshrc
+			# disable prompt while update
+			sed -i 's/# DISABLE_UPDATE_PROMPT="true"/DISABLE_UPDATE_PROMPT="true"/g' a.zsh
```
Comment (igorpecovnik) [inline debian-config-jobs:920]: Correct file name/location? Probably it should be /etc/skel/.zshrc ?

Comment (MalikSiraj) [reply debian-config-jobs]: corrected

**Comment by MalikSiraj** [comment]
what about "moreover, I think https://github.com/robbyrussell/oh-my-zsh.git should be changed to https://github.com/ohmyzsh/ohmyzsh.git although he is the creator of this still we should use a public repository"?

**Comment by igorpecovnik** [comment]
Yes, its safe to change URL to upstream - which is anyway a redirection.

**Pull request merged**